src/psmove.c

```diff
@@ -1715,19 +1784,35 @@ psmove_get_magnetometer(PSMove *move, int *mx, int *my, int *mz)
 {
     psmove_return_if_fail(move != NULL);
 
-    if (mx != NULL) {
-        *mx = TWELVE_BIT_SIGNED(((move->input.templow_mXhigh & 0x0F) << 8) |
-                move->input.mXlow);
-    }
+    if (move->model == Model_ZCM2) {
+        // NOTE: This model does not have magnetometers
```

Comment:
Should we disable/disallow magnetometer calibration in that case then?

Reply:
Yes, probably. But this touches on the whole calibration data topic that I did not (yet) want to get into since there are also changes in the associated HID report for the new PS4 Move controller (and probably in the actual calibration data itself).

Reply:
ok